examples/Learning.ipynb

```diff
@@ -0,0 +1,755 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Learning ML model on non-aggregated data"
+      ],
+      "metadata": {
+        "id": "Eult4u-UWW-D"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "OzIbbdhdia_s"
+      },
+      "source": [
+        "Welcome to this tutorial!\n",
+        "\n",
+        "We all know that data is required to train the model, and the quality of this data greatly affects the final result. Training a model on non-aggregated data usually leads to a loss in model quality, so the Toloka team has developed special layers for sublimating such data. In this notebook, we will solve this problem using the LSTM model and integrate the CoNAL and crowdlayer layers from Crowd-Kit library!"
```

Comment:
We have not developed them, we just implemented. Also `crowdlayer` -> `CrowdLayer`